sunpy/coordinates/transformations.py

```diff
@@ -197,11 +197,11 @@ def _observers_are_equal(obs_1, obs_2):
 
     # obs_1 != obs_2
     if obs_1 is None:
-        raise ConvertError("The source observer is set to None, but the destination observer is "
-                           f"{obs_2}.")
+        raise ConvertError("The source observer is None, but needs to be set as "
+                           f"the destination observer is {obs_2}.")
```

Comment:
My concern with this wording is it could be read as "you need to set the source observer to {obs_2}". Also I think "is set to None" is better, but it's very possible the user hasn't actually *set* the observer at all, i.e. if they are transforming to a frame class etc.

Reply:
How about:

"The source observer is set to None, but the transformation requires the source observer to be specified, as the destination observer is set to {obs_2}."?

Reply:
sounds good.